src/js/components/complete.jsx

```diff
@@ -18,7 +18,7 @@ class Complete extends Component {
           This meeting is now over. There is nothing left to see here. Move along.
         </div>
         <div className='oops'>
-          Didn't mean to end the meeting? <a href='#' onClick={this.goBack}>Go Back</a>
+          'Didn\'t mean to end the meeting? <a href='#' onClick={this.goBack}>Go Back</a>
```

Comment:
Using an apostrophe outside of a string is valid JSX, this doesn't need to be changed